client/packages/portal-client/src/framework-config/portal-framework-config.tsx

```diff
@@ -45,11 +46,29 @@ export function createPortalFramework(portalConfig: PortalConfig) {
 			});
 		}
 
-		config.onInitialized(async (fusion) => {
+		config.onInitialized<[NavigationModule]>(async (fusion) => {
 			configurePortalContext(fusion.context);
-			fusion.auth.defaultClient.setLogger(new ConsoleLogger(0));
+
+			fusion.context.currentContext$.pipe(skip(1)).subscribe((context) => {
+				const { navigator } = fusion.navigation;
+
+				if (!context) {
+					navigator.replace('/');
+				}
+
+				if (context && context.id && !window.location.pathname.includes(context.id)) {
+					const pathname = window.location.pathname.replace(
```

Comment:
Please extract the regex expression to a separate function and give the function a name explaining what the regex does